gui/builtinViews/fittingView.py

```diff
@@ -492,10 +492,11 @@ def spawnMenu(self):
         while sel != -1 and sel not in self.blanks:
             mod = self.mods[self.GetItemData(sel)]
 
-            # Test if mod.isEmpty exists.
-            try:
-                mod.isEmpty
-            except AttributeError:
+            # Test if mod.isEmpty does not exist.
+            # Certain special module can be missing this trait
+            # Example: T3D modes
+            if not hasattr(mod, 'isEmpty'):
+                # Set it if missing, prevents later stack traces.
```

Comment:
ha, I didn't mean set it if `hasattr` evaluated to false. I meant to not even assign the property `isEmpty` and use the `hasattr` like so:

`if hasattr(whatever) and not mod.isEmpty:`

This way we aren't setting a property that shouldn't be set. `isEmpty` applies to Modules only, not modes. No reason to have it around.